src/components/WalletAddress.js

```diff
@@ -169,8 +169,8 @@ export class WalletAddress extends React.Component {
     const showAddressString = () => {
       if (!LOCAL_STORE.isHardwareWallet()) {
         return (
-          <span ref="address" className="mt-1 mb-2">
-            {this.props.lastSharedAddress}
+          <span ref="address" className="mt-1 mb-2" id="currentAddressExhibitionWrapper">
+            <span id="currentAddressExhibitionSpan">{this.props.lastSharedAddress}</span>
```

Comment:
You could also try adding `data-happo-hide` (https://docs.happo.io/docs/hiding-content)

```suggestion
            <span data-happo-hide>{this.props.lastSharedAddress}</span>
```